nixos/lib/make-iso9660-image.sh

```diff
@@ -105,6 +105,7 @@ mkdir -p $out/iso
 # version-5 UUID's work)
 xorriso="xorriso
  -boot_image any gpt_disk_guid=$(uuid -v 5 daed2280-b91e-42c0-aed6-82c825ca41f3 $out | tr -d -)
+ -volume_date all_file_dates =1
```

Comment:
Didn't we start normalising epochs to 1980?

Reply:
The files in the ISO were mostly 1970 as they are copied from the store. The boot catalog file and root directory are 1980. I can change it to 1980 (or $SOURCE_DATE_EPOCH which `xorriso` says has value 315532800) here though if desired.

Reply:
Alright, it's changed to use SOURCE_DATE_EPOCH, which is 1980. Thanks for the tip.